Servers/UI/OJS.Servers.Ui/ClientApp/src/pages/shared/set-page-params.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { FC, useEffect } from 'react';
+import { useParams } from 'react-router';
+import isNil from 'lodash/isNil';
+
+import { IHaveChildrenProps } from '../../components/common/Props';
+import { useInternalUrlParams } from '../../hooks/common/use-internal-url-params';
+
+type IPageWithTitleProps = IHaveChildrenProps
+
+const Page = ({ children }: IPageWithTitleProps) => {
+    const params = useParams();
+    const { actions: { setParams } } = useInternalUrlParams();
+
+    useEffect(
+        () => {
+            if (isNil(params)) {
+                return;
+            }
+
+            setParams(params);
+        },
+        [ params, setParams ],
+    );
+
+    return (
+        <div>
```

Comment:
When I try to replace it with <></> the following ESLint Error happens: ESLint: Fragments should contain more than one child - otherwise, there’s no need for a Fragment at all.(react/jsx-no-useless-fragment). Any suggestions @Minkov?